imap_processing/hi/l1c/hi_l1c.py

```diff
@@ -102,14 +102,11 @@ def generate_pset_dataset(
     config_df = CalibrationProductConfig.from_csv(calibration_prod_config_path)
 
     pset_dataset = empty_pset_dataset(
+        de_dataset.epoch.data[0],
```

Comment:
I'm assuming that these will always be chronologically ordered, but if that's not guaranteed, use  `de_dataset.epoch.data.min()`?